style/americana.js

```diff
@@ -233,8 +233,14 @@ var map = (window.map = new maplibregl.Map({
   center: [-94, 40.5], // starting position [lng, lat]
   zoom: 4, // starting zoom
   attributionControl: false,
+  pitchWithRotate: false,
+  dragRotate: false,
+  touchPitch: false,
```

Comment:
Do we need to disable pitch?  It seems like this should still be okay in a north-up configuration.

Reply:
We don't necessarily _need_ to disable rotate or pitch  They're just a bit sub-optimal at the moment so I think it gives a better first impression to not have them enabled.  Leaving pitch enabled certainly looks less bad than the upside down shields you get with rotating, but it still looks rather odd to have the shields skewed to the plane of the map while place names and icons remain sticking up vertically.  

<img width="519" alt="Screen Shot 2022-01-28 at 1 48 05 PM" src="https://user-images.githubusercontent.com/281482/151604317-07d6d46c-5728-42fd-9f7d-f513e3b51dba.png">

If you'd like to keep the pitch enabled that seems fine to me.  I'd actually be fine with leaving rotate enabled too and not bothering with any of this (unless the issue never gets addressed in MapLibre).  Folks in Slack were just mentioning that it didn't look that great and I figured we could disable temporarily.

Reply:
Oh, ok I didn't realize pitch was borked too.  I'll add that to #111.

Reply:
So you want to move ahead with disabling both?  Or should we leave it as is for better visibility of why we want to see changes in MapLibre?

Reply:
Let's pose it as a community question.